packages/navigation-menu/src/lib/navigation-menu/components/menu-link/menu-link.component.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../assets/styles.scss";
```

Comment:
These are obviously not the styles from the styleguide, but from the navigation menu itself.

Reply:
Ok, I was mistaken since it's the only component that uses another separate `styles.scss`. However… I told you in advance that most of the styling will be moved to the core branding (so it can also be used in React later) and the way you implemented styling now is not quite 'migratable'. Can you propose a better way of implementing styling? If wanted, I can explain the way we did it before again.

Reply:
I renamed the file to variabels.scss, since that's what they are. I dont see any other issues with regards to migrating